### DISCUSSION TITLE
How to change number format when printing stresses ?

### DISCUSSION THREAD

**Discussion by Chivos** [robbievanleeuwen/section-properties#279] [Q&A] [answered]
Hi,
How can I change the format number of the stress scale when plotting the results ?
The idea is to format in float with only 1 or 2 decimals, which seems enough for stresses in steel sections when expressed in MPa.
I'am not used with python and matplotlib, and I can't found how.

**Comment by robbievanleeuwen** [comment, marked as answer]
Hi @Chivos, at the moment this is hardcoded, see this line [here](https://github.com/robbievanleeuwen/section-properties/blob/master/sectionproperties/analysis/section.py#L2821).

It seems like a good idea to provide this as an option - I will implement this in the upcoming `v3.0` with issue #280. In the meantime you can change this line in your installation of the library.

**Reply by Chivos** [reply]
Hi @robbievanleeuwen ,
Thanks for the information.

Can't wait for the V3, and thanks again for the incredible work done with sectionproperties !
